datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnectorTask.java

```diff
@@ -344,12 +345,24 @@ protected void postShutdownHook() {
       try {
         super.postShutdownHook();
       } catch (Exception e) {
-        postShutdownHookExceptionCaught = true;
+        _postShutdownHookExceptionCaught = true;
       }
     }
 
+    @Override
+    protected void seekToLastCheckpoint(Set<TopicPartition> topicPartitions) {
+      if (_failOnSeekToLastCheckpoint) {
+        throw new KafkaException("KafkaException: failed to seek");
+      }
+      super.seekToLastCheckpoint(topicPartitions);
+    }
+
+    void setFailOnSeekToLastCheckpoint() {
```

Comment:
nit: wondering if the setter can take true/false to allow for stop failing send failures also.

Reply:
done